mitodl/mit-open
# Improve offeror API performance

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4884

### Description (What does it do?)
Removes n+1 queries for offeror channel urls

### Screenshots (if appropriate):

![Screenshot 2024-07-16 125043](https://github.com/user-attachments/assets/91c3ce3f-7723-4443-ad56-e06d9a69297d)

### How can this be tested?
Hit the url http://api.open.odl.local:8063/api/v1/offerors at least a few times.  Check that:
  - channel_url is present and correct for each offeror
  - there are no n+1 query log warnings in the docker console
  - the response time for the api endpoint is good at http://api.open.odl.local:8063/silk/
